webroot/src/components/CollapseCaretButton/CollapseCaretButton.less

```diff
@@ -0,0 +1,9 @@
+//
```

Comment:
_UI testing find - just commenting here for a resolvable comment_

Also on the privilege-select screen, I noticed that the "Next" button becomes enabled as soon as the attestation modal appears. Through keyboard tab navigation it is possible to advance the form without completing the attestation. If this is an easy fix, you can address it here. If not, you can open a new ticket in the backlog that the team can prioritize in tomorrow's refinement.

<img width="802" alt="Screenshot 2024-11-11 at 10 27 35 AM" src="https://github.com/user-attachments/assets/04ebe434-e6b0-410e-bdaa-f4e92cfb86a3">

Reply:
I was aware of the button becoming enabled but I did not know this made caused a validation issue related to keyboard usage. Good catch

Reply:
Added to https://github.com/csg-org/CompactConnect/issues/297